src/test/ui/borrowck/borrowck-move-out-of-overloaded-auto-deref.mir.stderr

```diff
@@ -1,8 +1,10 @@
-error[E0507]: cannot move out of borrowed content
+error[E0507]: cannot move out of an `Rc`
   --> $DIR/borrowck-move-out-of-overloaded-auto-deref.rs:17:14
    |
 LL |     let _x = Rc::new(vec![1, 2]).into_iter();
-   |              ^^^^^^^^^^^^^^^^^^^ cannot move out of borrowed content
+   |              ^^^^^^^^^^^^^^^^^^^ cannot move out of an `Rc`
+   |
+   = help: you may wish to consider `Rc::try_unwrap`?
```

Comment:
Sorry for the wild goose chase, but now I'm worrying that this will mislead "newcomers" — `try_unwrap` is probably *very rarely* the thing you actually want. I've heard time and time again feedback that having the compiler give "random" suggestions can be very misleading for people who are feeling their way through the system. Perhaps we should just remove this. 

r=me with `try_unwrap` removed

Reply:
Removed that commit.